Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -559,6 +559,130 @@ final class WorkspaceTests: XCTestCase {
             }
         }
     }
+    
+    func testPackageEditAndRemove() {
+        // We mock up the following dep graph:
+        //
+        // Root
+        // \ A: checked out (@v1)(in editable mode)
+        // Then will update to:
+        // Root
```

Comment:
Please remove these comments, they tend to get out of date. There are some similar comments in these tests, they need to be cleaned up at some point.